tests/bal-tools-integration-tests/src/test/resources/projects/projectWithOneToolWithDepsTomlAndSticky/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "asmaj"
```

Comment:
Shall we use a generic naming for org

Reply:
Addressed in [0051112](https://github.com/ballerina-platform/ballerina-lang/pull/44104/commits/0051112995b8aa8fd9bb7df0099b63fe94e3a5b1)